src/styles/main.scss

```diff
@@ -1,3 +1,83 @@
+$day-size: 100px;
+$day-border-width: 1px;
+$font-size: 30px;
+$padding: 10px;
+$day-gap: 1px;
+$day-bg-color: #eee;
+$text-color: #000;
+$font-family: Arial, sans-serif;
+$hover-color: #FFBFCB;
+$transition-duration: 0.5s;
+$week-days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
 body {
   margin: 0;
 }
+
+.calendar {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  width: (7 * $day-size) + (6 * $day-gap);
+  background-color: white;
+
+}
+
+  .calendar__day {
+    width: $day-size;
+    height: $day-size;
+    border: $day-border-width solid black;
+    box-sizing: border-box;
```

Comment:
it s better to apply this property to all elements (use `*` selector)

When all elements have the same box-sizing value, their behavior becomes more predictable